PSReadLine/Keys.cs

```diff
@@ -459,6 +459,7 @@ internal static bool IgnoreKeyChar(this ConsoleKeyInfo key)
                 case ConsoleKey.F22:
                 case ConsoleKey.F23:
                 case ConsoleKey.F24:
+                case ConsoleKey.Backspace:
```

Comment:
I don't think it matters because I intend to make `Backspace`, `Ctrl+h` and `Ctrl+Backspace` behave the same on Linux -- all interpreted to be `Backspace`.

On Ubuntu, `Ctrl+h` and `Ctrl+Backspace` return the exact same `ControlKeyInfo` -- `(keychar: 0x08, key: Backspace, mods: 0)`, while `Backspace` returns `(keychar: 0x7f, key: Backspace, mods: 0)`. With this change, all three chords will be treated as `Backspace`, which is what Bash does already (at least on Ubuntu 16.04 and 14.04).

Reply:
`Ctrl+h` and `Ctrl+Backspace` result in the same `ControlKeyInfo` on Linux. If you want `Ctrl+Backspace` to do the same as on Windows (`BackwardKillWord`), then there is no way to fix #619 because pressing `Ctrl+h` on Linux has the same effect of `Ctrl+Backspace` being pressed.

Do you want to close #619 as by design then?

Reply:
> because we can distinguish Backspace from Ctrl+Backspace, I think we want to continue to be able to do that.

`PSReadline` only worked on `Windows` at that time, and thus that request totally made sense. In fact, this PR doesn't break `Ctrl+Backspace` on Windows, and it would work exactly the same as before.

Now that `PSReadline` works on Linux and macOS too, I think we should respect the special default key bindings like `Ctrl+h` and `Ctrl+Backspace` on those platforms.

Today, `Ctrl+Backspace` doesn't work as expected (`BackwardKillWord`) on Linux for both 1.2 and 2.0 versions; it simply behaves as `Backspace` (hence this PR doesn't break `Ctrl+Backspace` on Linux too). Do you know of any complaints about its current behavior on Linux/macOS? If not, then I think it probably makes sense to comply with the bash defaults.

Reply:
I'm suggesting that `Ctrl+Backspace` can work as users expect for their preferred bindings (`Windows` or `Emacs` or `Vi`) on any OS, but not with this change.

Basically, I'm suggesting this matrix will make the most number of people happy and won't be difficult to support:

|       Key      | Windows Mode on Windows |     Emacs Mode     | Windows Mode on Linux |
|:--------------:|:-----------------------:|:------------------:|:---------------------:|
|    Backspace   |    BackwardDeleteChar   | BackwardDeleteChar |   BackwardDeleteChar  |
| Ctrl+Backspace |    BackwardKillWord   | BackwardDeleteChar |   BackwardKillWord  |
|     Ctrl+h     |    BackwardDeleteChar   | BackwardDeleteChar |   BackwardKillWord  |